src/templates/static/legal/metatrader/terms_and_conditions.jsx

```diff
@@ -1,446 +1,660 @@
+                    <li>{it.L('Given the Company\'s role as a Margin trading service provider as described in this agreement, the Company seeks to avoid undue market influence to the extent consistent with the Client\'s trading needs and the Company\'s risk management policies and procedures. By continuing to use the Margin trading services described in this agreement, the Client acknowledges that they are aware of the potential conflict of interest disclosed that may arise and cannot be completely eliminated, and they consent to the Company acting notwithstanding such potential conflict of interests.')}</li>
                 </ol>
             </li>
-            <li>{it.L('Client Money')}
+            <li>{it.L('Client money')}
                 <ol>
-                    <li>{it.L('Money collected from Clients are not invested in any securities, futures, or other investments, on behalf of Clients.')}</li>
-                    <li>{it.L('The company holds customer funds in separate bank accounts to the operational accounts; and arrangements have been made to ensure assets in the customer accounts are distributed to customers in the event of insolvency.')}</li>
-                    <li>{it.L('Your account should not be used as a banking facility and deposits should only be made with a view to using funds to place contracts. The Company is not a financial institution. Clients will not receive interest on deposits. Should you make repeated deposits and withdrawals without commensurate contracts being placed, we reserve the right to pass on to your accounts, without prior notice, any bank charges we have incurred before closing the account.')}</li>
+                    <li>{it.L('The money collected from Clients is not invested in any securities, futures, or other investments, on behalf of Clients.')}</li>
+                    <li>{it.L('The Company holds customer funds in bank accounts separate from the operational accounts, and arrangements have been made to ensure that the assets in the customer accounts will be distributed to the customers in the event of insolvency.')}</li>
+                    <li>{it.L('The Client\'s account should not be used as a banking facility, and deposits should only be made with a view to using funds for placing contracts. The Company is not a financial institution. Clients will not receive interest on deposits. Should the Client make repeated deposits and withdrawals without commensurate contracts being placed, the Company reserves the right to pass on to the Client\'s account, without pr
```

Comment:
```suggestion
                    <li>{it.L('The personal information that the Company collects and holds is treated as confidential. Confidential information will not be used for any purpose other than in connection with the provision of the Margin trading services described in this agreement. Confidential information will only be disclosed as and when required by law. The Company may also disclose such personal information to: (i) other companies within the Binary Group of companies; (ii) any such third parties as the Company sees fit to assist in servicing the Client\'s MT5 Real Account; (iii) regulatory authorities upon their reasonable request; (iv) any such third parties as the Company sees fit to assist in enforcing the Company\'s legal or contractual rights against the Client, including but not limited to legal advisors; (v) third parties as the Company sees fit to investigate or prevent fraud or other illegal activities.')}</li>
```

Reply:
For some reason the above correction doesn't show the right sentence that needs replacing (line 258). It's all right in the Preview, but when I add the comment, it gets like this.